docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport Database Access
+---
+
+Access to AWS DynamoDB can be provided by [Teleport Database
+Access](../introduction.mdx). This allows for
+fine-grain access control through [Teleport's RBAC](../../database-access/rbac.mdx).
+
+This guide will help you to:
+
+- Install the Teleport Database Service.
+- Set up the Teleport Database Service to access DynamoDB.
+- Connect to your DynamoDB databases through the Teleport Database Service.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![DynamoDB Self-Hosted](../../../img/database-access/guides/aws-dynamodb_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![DynamoDB Cloud](../../../img/database-access/guides/aws-dynamodb_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- A running Teleport cluster. If you don't already have Teleport running, see
+  [How to Choose a Teleport Edition](../../choose-an-edition/introduction.mdx)
+  to learn more about your options, and how to set up Teleport.
+- AWS account with DynamoDB databases.
+- IAM permissions to create IAM roles.
+- `aws` Command Line Interface (CLI) tool installed in PATH.
```

Comment:
```suggestion
- `aws` Command Line Interface (CLI) tool installed in `$PATH`.
```

Do you think this makes it more obvious that we're referring to an environment variable? Feel free to ignore this comment if you don't think so.

Reply:
I have no strong opinion either way, so happy to oblige.